tcms/testruns/templates/testruns/mutable.html

```diff
@@ -36,25 +36,25 @@
             </div>
             <div class="col-md-3 col-lg-3">
                 <select class="form-control selectpicker" id="id_build" name="build">
-                {% for option in form.build.field.queryset %}
-                    <option value="{{ option.pk }}" {% if option.pk|escape == form.build.value|escape %}selected{% endif %}>{{ option.name }}</option>
+                {% for option in builds %}
+                    <option value="{{ option.pk }}" {% if option.pk|escape == object.build_id|escape %}selected{% endif %}>{{ option.name }}</option>
                 {% endfor %}
                 </select>
                 <p class="help-block">
-                    {% trans "Product" %}: {{ test_plan.product.name }}, {% trans "version" %} {{ test_plan.product_version.value }}
+                    {% trans "Product" %}: {{ test_plan.product.name }}, {% trans "version" %} {{ test_plan.product_version }}
                 </p>
             </div>
 
             <label class="col-md-1 col-lg-1" for="id_manager">{% trans "Manager" %}</label>
             <div class="col-md-3 col-lg-3 {% if form.manager.errors %}has-error{% endif %}">
-                <input id="id_manager" name="manager" value="{{ form.manager.value }}" type="text"
+                <input id="id_manager" name="manager" value="{{ object.manager }}" type="text"
                     class="form-control" placeholder="{% trans 'Username or email' %}" required>
                 {{ form.manager.errors }}
             </div>
 
             <label class="col-md-1 col-lg-1" for="id_default_tester">{% trans "Default tester" %}</label>
             <div class="col-md-3 col-lg-3 {% if form.default_tester.errors %}has-error{% endif %}">
-                <input id="id_default_tester" name="default_tester" value="{{ form.default_tester.value|default:'' }}" type="text"
+                <input id="id_default_tester" name="default_tester" value="{{ object.default_tester|default:'' }}" type="text"
```

Comment:
Why change and remove `.value` ?

Reply:
Just preferred to use the model iteself.